README.md

```diff
@@ -1,25 +1,17 @@
-> ⚠️ The [CLI branch](https://github.com/scaffold-eth/scaffold-eth-2/tree/cli) is under active development.  
-> If you find any bug, please report as [issue](https://github.com/scaffold-eth/scaffold-eth-2/issues) or send a message in [🏗 scaffold-eth developers chat](https://t.me/joinchat/F7nCRK3kI93PoCOk)
+> ⚠️ Under active development.
+>
+> If you find any bug, please report as [issue](https://github.com/scaffold-eth/create-eth/issues) or send a message in [🏗 scaffold-eth developers chat](https://t.me/joinchat/F7nCRK3kI93PoCOk)
 
-# 🏗 Scaffold-ETH 2
+# 🏗 create-eth
+
+CLI to create decentralized applications (dapps) using Scaffold-ETH 2.
```

Comment:
What about we add here something like:

«This is an alternative method of installing Scaffold-ETH. Instead of directly cloning SE-2 (link), you can use create-eth to create your own custom instance, where you can choose among several configurations and extensions»

Maybe not the best phrasing haha, but I think it's important to make it clear.

Open to suggestions!!

P.S. We could also write something similar in the SE-2 repo, pointing to this one.

Reply:
> Maybe not the best phrasing haha, but I think it's important to make it clear.

Yup agree, also the desc you mentioned already looks nice to me !!

Reply:
I like the explanation! Just pushed it 🙌